.github/CONTRIBUTING.md

```diff
@@ -129,18 +129,18 @@ Additional references:
   If the changes are related to an existing GitHub issue,
   please reference the issue in PR description (e.g. ```Fix #11```).
   See [this][closing-via-message] for more details.
-* If the change warrants a note in the [changelog](../CHANGELOG.MD)
-  either update the changelog in your pull request or
-  add a comment in the PR description saying that the change may warrant a note in the changelog.
-  New changes always go into the **Unreleased** section.
+
+#### Pull request - Change log
+
+* All PRs must either update the [changelog](../CHANGELOG.MD) in your pull request.
```

Comment:
`either` is either inappropriate or part of this sentence is missing

Reply:
resolved